updated screenshots for managing_pipelines and quick_pipeline_setup

adityasood opened the pull request.

arvindsv commented:
@adityasood Wow! Looks like it must have taken a lot of time. :) Some thoughts:

In [this](https://github.com/adityasood/docs.go.cd/blob/f900e82ee72ef62a64fb418c2275cab8c193e980/resources/images/topnav_admin.png): Focus on "Pipelines" rather than "User Summary", since that is what is used next in /configuration/admin_use_custom_pipeline_label.html

In [this](https://github.com/adityasood/docs.go.cd/blob/f900e82ee72ef62a64fb418c2275cab8c193e980/resources/images/create_new_pipeline_link.png) and [this](https://github.com/adityasood/docs.go.cd/blob/f900e82ee72ef62a64fb418c2275cab8c193e980/resources/images/edit_pipeline_link.png): Images have been stretched. See in /configuration/managing_pipelines.html

In [this](https://github.com/adityasood/docs.go.cd/blob/f900e82ee72ef62a64fb418c2275cab8c193e980/resources/images/pipeline_templates.png): Because the order of the numbering of the arrows has changed, the "Key" in the "Pipeline templates" section in /navigation/administration_page.html#pipeline-templates is wrong.

In [this](https://github.com/gocd/docs.go.cd/pull/188/files?diff=split&short_path=6502fcc#diff-6502fccb910fd0c1e5e16911f3950eb9): Some images seem too big. For instance, this one changed from 431px to 3163px. You'll need to make it smaller and run optipng on it.

adityasood commented:
@arvindsv I have made the following changes

1. Changed the image [here](https://github.com/adityasood/docs.go.cd/blob/92588cf60c6c64c89fcabc67ba03bb86bde7bb90/resources/images/topnav_admin.png) to highlight pipelines.

2. Changed the stretched images [here](https://github.com/adityasood/docs.go.cd/blob/92588cf60c6c64c89fcabc67ba03bb86bde7bb90/resources/images/create_new_pipeline_link.png) and [here](https://github.com/adityasood/docs.go.cd/blob/92588cf60c6c64c89fcabc67ba03bb86bde7bb90/resources/images/edit_pipeline_link.png) 

3. Changed the order of numbering [here](https://github.com/adityasood/docs.go.cd/blob/92588cf60c6c64c89fcabc67ba03bb86bde7bb90/navigation/administration_page.md) to match the image [here](https://github.com/adityasood/docs.go.cd/blob/92588cf60c6c64c89fcabc67ba03bb86bde7bb90/resources/images/pipeline_templates.png)

4. Ran optipng on all images in the latest commit.

5. Have been unable to reduce the size of the images as the text was getting distorted and the images didnt look good.
